docs-v2/integrations/all/microsoft.mdx

```diff
@@ -55,9 +55,9 @@ const { data } = await nango.createConnectSession({
   integrations_config_defaults: {
     "microsoft": {
       authorization_params: {
-        response_type: 'code'
-      },
-      default_scopes: '.default'
+        response_type: 'code',
+        scope: '.default'
+      }
```

Comment:
NB: this snippet has been copy pasted on all aliases of microsoft

Reply:
thanks, will update